src/App-au.vue

```diff
@@ -58,6 +60,7 @@ export default defineComponent({
     if (responseData) {
       this.terms = responseData.terms
       this.tabs = getTabsData(responseData.products)
+      this.cards = getCardsData(responseData.cards)
     }
   },
 })
```

Comment:
`cards` data is required because card image is shown in the overlay now 👇 
![image](https://user-images.githubusercontent.com/63985172/134828207-b8ec6fe9-de9b-402a-b57d-357f1bdb08b3.png)